fabfile.py

```diff
@@ -724,35 +724,43 @@ def bootstrap(projectpath):
 
 
 @task
-def bootstrap_from_checkout():
+def bootstrap_from_checkout(backup=False):
     """
     Creates the virtualenv and install the app from git checkout
     """
-    execute(updatemaincode)
+    execute(updatemaincode, backup=backup)
     execute(build_virtualenv)
-    execute(app_update_dependencies)
+    execute(app_update_dependencies, backup=backup)
     execute(app_setup)
     execute(check_and_create_database_user)
     execute(app_compile_nodbupdate)
     execute(set_file_permissions)
-    execute(app_db_install)
+    if not backup:
+        execute(app_db_install)
+    else:
+        execute(database_restore, backup=backup)
     execute(app_reload)
     execute(webservers_reload)
 
 
 @task
-def bootstrap_from_backup():
+def bootstrap_from_backup(backup=True):
     """
     Creates the virtualenv and install the app from the backup files
     """
-    execute(app_update_dependencies_for_backup)
-    execute(app_setup)
-    execute(check_and_create_database_user)
-    execute(app_compile_nodbupdate)
-    execute(set_file_permissions)
-    execute(backup_database_restore)
-    execute(app_reload)
-    execute(webservers_reload)
+    execute(bootstrap_from_checkout, backup=backup)
+
+
+@task
+def install_assembl_requirements():
```

Comment:
This task is mentioned nowhere. We don't know when we have to run it. If you add it, mention it in documentation (INSTALL.md ?)